Client/game_sa/CVehicleSA.cpp

```diff
@@ -1333,37 +1333,37 @@ void CVehicleSA::RecalculateHandling()
     // Put it in our interface
     CVehicleSAInterface* pInt = GetVehicleInterface();
     unsigned int         uiHandlingFlags = m_pHandlingData->GetInterface()->uiHandlingFlags;
-    bool                 hydralicsInstalled, nitroInstalled;
+    bool                 hydralicsInstalled = false, nitroInstalled = false;
 
     // We check whether the user has not set incorrect flags via handlingFlags in the case of nitro and hydraulics
     // If this happened, we need to correct it
-    for (auto& upgradeID : pInt->m_upgrades)
+    for (const std::int16_t upgradeID : pInt->m_upgrades)
     {
         // Empty upgrades value is -1
         if (upgradeID < 0)
             continue;
 
         // If NOS is installed we need set the flag
-        if ((upgradeID >= 1008 && upgradeID <= 1010) && uiHandlingFlags | HANDLING_NOS_Flag)
+        if ((upgradeID >= 1008 && upgradeID <= 1010) && (uiHandlingFlags | HANDLING_NOS_Flag))
```

Comment:
what is `uiHandlingFlags | HANDLING_NOS_Flag` supposed to do in your opinion?

Reply:
I was relying on the default code and didn't even notice this mistake, ty